symengine/printer.cpp

```diff
@@ -362,6 +362,44 @@ void StrPrinter::bvisit(const UnivariatePolynomial &x) {
     str_ = s.str();
 }
 
+void StrPrinter::bvisit(const UnivariateSeries &x) {
+    std::ostringstream o;
+    bool first = true;
```

Comment:
You should simply use the printing method of `UnivariateExprPolynomial` and then add the `O` term. (`O(x**10)`)